azurerm/internal/services/sentinel/sentinel_alert_rule_fusion_resource_test.go

```diff
@@ -0,0 +1,167 @@
+package sentinel_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/sentinel/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+type SentinelAlertRuleFusionResource struct{}
+
+func TestAccAzureRMSentinelAlertRuleFusion_basic(t *testing.T) {
```

Comment:
And removing AzureRM from the test names
```suggestion
func TestAccSentinelAlertRuleFusion_basic(t *testing.T) {
```

Reply:
updated